.stylelintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   extends: "stylelint-config-standard",
   customSyntax: "postcss-scss",
-  ignoreFiles: ["war/src/main/scss/_bootstrap.scss"],
+  ignoreFiles: ["src/main/scss/_bootstrap.scss"],
```

Comment:
Part of the Webpack build invoked from the root Maven module, so the source file moves to the root Maven module.